pycheribuild/config/compilation_targets.py

```diff
@@ -611,7 +611,7 @@ def essential_compiler_and_linker_flags(self) -> typing.List[str]:
 
     @property
     def must_link_statically(self):
-        return True  # dynamic linking is still experimental
+        return False
```

Comment:
This is already the default in the parent class, so you could just remove the override.